### PR TITLE
Feat post submit survey page

### PR DESCRIPTION
**Pull request by jdirand**
- Add post submit survey page
- Update survey for metro and add dom survey

### CODE REVIEW

File: src/components/postSubmit/AdditionalInformation.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+}
+
+  function FeedbackLink({href, imageSrc, label}: {href:string, imageSrc:string, label:string}) {
+    if (href && imageSrc) {
+      return (
+        <a title={`${label} - ouvre une nouvelle fenêtre`} href={href}>
+          <img src={imageSrc} alt="" />
+        </a>
+      )
+    } else if (href) {
```
Comment (davdarras) [inline src/components/postSubmit/AdditionalInformation.tsx:20]: else could be deleted here

Comment (isMattCoding) [reply src/components/postSubmit/AdditionalInformation.tsx:20]: I would suggest an early return of null : 
```js 
if(!href) {
return null;
} 
if(href && imageSrc) { return (<a> with imageSrc)} 
return (<a> without imageSrc)
```
And agree, no need for else if for last return

File: src/components/postSubmit/AdditionalInformation.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        </a>
+      )
+    }
+    return null;
+  }
+  function ProvideFeedback({submit, Feedback}: {submit: SubmitType, Feedback: LinkType}){
+    const {link, title} = Feedback;
+    const {href, imageSrc, label} = link
+
+    if (submit && Feedback) {
```
Comment (isMattCoding) [inline src/components/postSubmit/AdditionalInformation.tsx:33]: Same thing here: 
```js 
if (!submit && !Feedback) {
return null; 
} 
return ( ... ) 
```

File: src/components/postSubmit/PostSubmitSurveyType.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+export type WelcomeType = {
```
Comment (isMattCoding) [inline src/components/postSubmit/PostSubmitSurveyType.ts:1]:
```js
export type PostSubmitSurveyType
```

**Pull request merged**